File: lua/refactoring/tests/refactor/119/lua/custom_extract_var_statement/extract_var.expected.lua
-- stylua: ignore start

local function orderCalculation(order, a, b)
    local temp = a * b
    local basePrice = order.quantity * order.itemPrice -- poggers!
    local blah = (basePrice) - 7

    print(string.format("%d %d"), temp, blah)

    return basePrice
        - math.max(0, order.quantity - 500) * order.itemPrice * 0.05
        + math.min(basePrice * 0.1, 100)
end

File: lua/refactoring/tests/refactor/119/cpp/custom_extract_var_statement/extract_var.expected.cpp
#include <algorithm>
#include <iostream>

struct Order {
  double quantity;
  double itemPrice;
};

double orderCalculation(Order order, int a, int b) {
  float temp = a * b;
  const auto basePrice = order.quantity * order.itemPrice;
  float blah = (basePrice) - 7;

  std::cout << temp << " " << blah << std::endl;

  return basePrice -
         std::max(0.0, order.quantity - 500) * order.itemPrice * 0.05 +
         std::min(basePrice * 0.1, 100.00);
}
